test/revert.spec.ts

```diff
@@ -176,7 +177,7 @@ describe(`Reverts (Seaport v${VERSION})`, function () {
               value,
             }
           )
-      ).to.be.revertedWith("BadFraction");
+      ).to.be.revertedWithCustomError(marketplaceContract, "BadFraction");
```

Comment:
`revertedWithCustomError` needs a reference to the contract that defines the custom error